apps/voting/contracts/Voting.sol

```diff
@@ -58,14 +62,19 @@ contract Voting is IForwarder, AragonApp {
     mapping (uint256 => Vote) internal votes;
     uint256 public votesLength;
 
+    uint64 public overruleWindow;
+    mapping (address => mapping (address => bool)) internal representatives;
```

Comment:
If we finally decide not to allow having multiple representatives, I guess we could shorten this double mapping.

Reply:
Sure we can have simply `mapping (address => address)`